Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -431,14 +431,46 @@
     // FIXME: We should also set the openInPreviewIndex when UnifiedPdfPlugin::openWithPreview is implemented
     menuItems.append({ WebCore::contextMenuItemPDFOpenWithPreview(), true, false, 0, true, static_cast<uint8_t>(ContextMenuItemTag::OpenWithPreview) });
 
+    auto currentDisplayMode = contextMenuItemTagFromDisplyMode(m_documentLayout.displayMode());
+    // Single page
+    menuItems.append({ WebCore::contextMenuItemPDFSinglePage(), true, false, currentDisplayMode == ContextMenuItemTag::SinglePage, true, static_cast<uint8_t>(ContextMenuItemTag::SinglePage) });
```

Comment:
Please call `enumToUnderlyingType()` instead of an explicit static_cast<> like this which could get out-of-sync with the actual underlying type.

Same comment applies below.

Reply:
Done!